---
layout: page
title: Projects
subtitle: The interesting part
---

## Current projects:

Under Keith’s mentorship, I led and conducted a project to study the role of a DYRK kinase (TbDYRK) in the quorum-sensing response of Trypanosoma brucei. I developed a pipeline for a systematic analysis of the effect of point mutations and deletions of key residues or regions of interest for the protein kinase function and on the phenotype of differentiation of the parasite. In more detail, I developed 2 different types of in-gel kinase assay. The first one is based on the use of a thio-ester-modified phosphate on ATP[^1] that can be revealed by an antibody, whereas the second system is based on the radioactive labelling kinase assay I developed during my PhD thesis. Both gel methods were transferred for western blotting, enabling the study of the kinase activity of different mutants purified from an insect expression system. The description of these 2 in-gel kinase assays has been published in Methods in Molecular Biology (Clifton, N.J.) in 2020[^2]. I then performed an *in vivo* analysis of the effect of the mutations on the phenotype of *T. brucei* *in vitro* slender to stumpy differentiation[^3]<sup>,</sup>[^4]. These enabled me to identify key residues for the kinase regulation and key functional residues for the function of TbDYRK that were evolutionary divergent compared the those classically found in this protein kinase family in other model organisms.

The phosphoproteome of a TbDYRK null mutant was then compared to parental cells by quantitative mass spectrometry using TMT isobaric labelling to identify the pathways in which TbDYRK is implicated. As a complementary approach, I also performed a kinase assay on parasite lysates using a non-mutated and an inactive mutant of the kinase, purified from the insect cell system. These complementary approaches, followed by generation of knock-downs (using RNAi) and knock-outs (using CrispR/Cas9), confirmed direct interaction of effectors with TbDYRK and their involvement in the quorum-sensing differentiation. For example, the negative regulator of transcription NOT5 was identified as a substrate of TbDYRK potentially inactivated by phosphorylation. Furthermore, NOT5 knock-down led parasites to differentiate prematurely in mice. Conversely, TbDYRK phosphorylates the zinc finger protein ZC3H20, an mRNA regulator that promotes for stumpy formation, as confirmed by the inability of the knocked-out cell line to differentiate. These results suggest that TbDYRK can both inhibit ‘slender retainer’ proteins and activate ‘stumpy inducer’ molecules by phosphorylation, driving differentiation (model presented in [Figure 1](#figure-1)). To decipher in more details the regulation role played by the kinase TbDYRK up to a single amino acid, I identified the phosphorylated residues on one of its substrates, ZC3H20. I then performed endogenous single point mutations, using an adapted version of our CrispR system, to eliminate the phosphorylable amino acids. Those mutations led to a complete incapacity of the parasite to differentiate from slender to stumpy form *in vitro* and *in vivo*. These results confirmed the identification of the pathway, from the signal induction, leading to the activation mechanisms of the TbDYRK, which in turn phosphorylates and activates ‘stumpy inducers’ proteins such as ZC3H20 and also phosphorylates and inhibits ‘slender retainer’ proteins such as NOT5. All these results were included in a publication in eLife in 2020[^5].

#### Figure 1: 

{: .box-figure}
![Working model TbDYRK](/assets/img/model2.png){: .mx-auto.d-block :}
*Model of the activation mechanisms and function of TbDYRK. A- Phosphorylation of the pre-active kinase in response to the stumpy inducing signal results in activation of the kinase. b- Consequences of the activation of TbDYRK for the regulation of differentiation, through the inactivation of slender retainer molecules and the activation of stumpy inducers.*

One particular observation I made was that amongst the list of potential substrates of TbDYRK, a large number were associated with mRNA regulation functions and potentially implicated in stress granules formation. This led me to the project that I am currently developing which consists of the observation and analysis of the regulation of stress granules in *T. brucei* during parasite differentiation. For this project, that requires in-situ tagging and imaging, I supervised another Honours student, Eliza Waskett, who was rewarded with the SBS Honours Prize 2020 for her work under my supervision. We were able to determine that particular components of previously identified stress granules[^6] were dynamically targeted to different components during the course of *in vivo* differentiation. I am also developing a proximity labelling system using the APEX2 tag to allow the identification of new components of stress granules formed during the quorum sensing differentiation, as we observed that their composition is different from the granules generated during glucose starvation. In parallel, and in order to filter the large list of potential substrates implicated in stress granule formation that we obtained from the TbDYRK analysis, I initiated and led a mini bioinformatic project during the lockdown imposed by the current COVID outbreak that aimed to identify low complexity regions (LCRs) in the proteome of *T. brucei*. Indeed, the formation of stress granules occurs through liquid-liquid phase separation of the proteins that is facilitated by the presence of low complexity regions. We were able to determine an enrichment of LCRs in protein’s extremities and particularly on the C-terminal regions of proteins involved in nucleic acid binding. I then linked the regulation of these regions to post-translational modifications and was able to show that phosphorylations are strongly enriched in LCRs. These results open the way to study the regulations of stress granules formation by the action of kinases and phosphatases in kinetoplastids parasites and were released in a publication in Open Wellcome research in 2020[^7]. 

As part of a collaborative project led by Dr. Lindsay McDonald, I analysed results of a quantitative phosphoproteomic study and identified a list of potential substrates of the *T. brucei* MEKK1 protein kinase implicated in the slender to stumpy differentiation. The phosphoproteomic analysis identified molecules with altered expression and phosphorylation profiles in MEKK1 null mutants, including another component in the quorum-sensing pathway, NEK17. This collaboration was rewarded by a publication in PLOS Pathogen in 2018[^8].

I also contributed to the project initiated by my co-worker Dr. Federico Rojas for the development of a trypanosome *in vitro* differentiation system[^9]. The successful development and release to the community of this *in vitro* system will allow reduction of animal use by the trypanosome research community. 

### [Past projects](/pastprojects)
#### Bibliography

[^1]: Allen, J. J. et al. A semisynthetic epitope for kinase substrates. Nat. Methods 4, 511–516 (2007).
[^2]: Szöőr, B. & Cayla, M. Gel-Based Methods for the Investigation of Signal Transduction Pathways in Trypanosoma brucei. in Methods in molecular biology (Clifton, N.J.) (eds. Michels, P. A. M., Ginger, M. L. & Zilberstein, D.) vol. 2116 497–522 (NLM (Medline) - Springer US, 2020).
[^3]: MacGregor, P. & Matthews, K. R. Identification of the regulatory elements controlling the transmission stage-specific gene expression of PAD1 in Trypanosoma brucei. Nucleic Acids Res. 40, 7705–7717 (2012).
[^4]: Laxman, S., Riechers, A., Sadilek, M., Schwede, F. & Beavo, J. A. Hydrolysis products of cAMP analogs cause transformation of Trypanosoma brucei from slender to stumpy-like forms. Proc. Natl. Acad. Sci. U. S. A. 103, 19194–19199 (2006).
[^5]: Cayla, M., McDonald, L., MacGregor, P. & Matthews, K. An atypical DYRK kinase connects quorum-sensing with posttranscriptional gene regulation in Trypanosoma brucei. Elife 9, 1–33 (2020).
[^6]: Fritz, M. et al. Novel insights into RNP granules by employing the trypanosome’s microtubule skeleton as a molecular sieve. Nucleic Acids Res. 43, 8013–8032 (2015).
[^7]: Cayla, M., Matthews, K. R. & Ivens, A. C. A global analysis of low-complexity regions in the Trypanosoma brucei proteome reveals enrichment in the C-terminus of nucleic acid binding proteins providing potential targets of phosphorylation. Wellcome Open Res. 5, 219 (2020).
[^8]: McDonald, L. Cayla, M., Ivens, A., Mony, B., MacGregor, P., Silvester, E., McWilliam, K., Matthews, K., Non-linear hierarchy of the quorum sensing signalling pathway in bloodstream form African trypanosomes. PLoS Pathog. 14, e1007145 (2018).
[^9]: Rojas, F., Cayla, M., Matthews, K. Basement membrane proteins as a substrate for efficient trypanosoma brucei differentiation *in vitro*. PLoS Neglected Tropical Diseases 15, e0009284 (2021).
 

